content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Building your own Collector instance
+weight: 30
+---
+
+If you are planning to build and debug custom collector receivers, processors,
+extensions, or exporters, you are going to need your own Collector instance.
+That will allow you to launch and debug your OpenTelemetry Collector components
+directly within your favorite Golang IDE.
+
+The other interesting aspect of approaching the component development this way
+is that you can use all the debugging features from your IDE (stack traces are
+great teachers!) to understand how the Collector itself interacts with your
+component code.
+
+The OpenTelemetry Community developed a tool called [OpenTelemetry Collector
+builder][ocb] (or `ocb` for short) to assist people in assembling their own
+distribution, making it easy to build a distribution that includes their custom
+components along with components that are publicly available.
+
+As part of the process the `builder` will generate the Collector's source code,
+which you can use to help build and debug your own custom components, so let's
+get started.
+
+## Step 1 - Install the builder
+
+The `ocb` binary is available as a downloadable asset from the [OpenTelemetry
+Collector releases page][releases]. You will find the list of assets at the
+bottom of the page.
+
+v0.53.0 is the latest and the assets are named based on OS and chipset, so
```

Comment:
You can introduce the collector version as a variable, see https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/collector/getting-started.md?plain=1 how you can do that.

Reply:
I'm willing to tackle this separately quite soon -- via #1194. Would that be ok?

Reply:
I am ok with that